.github/scripts/Invoke-LibraryUpdatePolicyAssignments.ps1

```diff
@@ -61,7 +61,7 @@ $defaultParameterValues =@(
     "-p automationRegion=`${default_location}",
     "-p retentionInDays=30",
     "-p rgName=`${root_scope_id}-mgmt",
-    "-p logAnalyticsResourceId=/subscriptions/00000000-0000-0000-0000-000000000000/resourcegroups/`${root_scope_id}-mgmt/providers/Microsoft.OperationalInsights/workspaces/`${root_scope_id}-la",
+    "-p logAnalyticsResourceId=/subscriptions/00000000-0000-0000-0000-000000000000/resourcegroups/`${root_scope_id}-mgmt/providers/microsoft.operationalinsights/workspaces/`${root_scope_id}-la",
```

Comment:
Hi @jaredfholgate 

If we change the case for all assignments, as appears to be the case here - would all assignments that use this value be re-created?

Reply:
This parameter is literally only used by that one policy, so there is no impact outside of that one. That one policy is an outlier, all others construct the id based on parameters, whereas this one takes the whole id as a parameter.